src/olympia/addons/models.py

```diff
@@ -1177,11 +1177,7 @@ def get_icon_url(self, size, use_default=True):
             return jinja_helpers.user_media_url('addon_icons') + path
 
     def get_default_icon_url(self, size):
-        # We don't update the default icon very frequently so there is no
-        # automated query parameter for cachebusting...
-        return '{}img/addon-icons/{}-{}.png?v=20210601'.format(
-            settings.STATIC_URL, 'default', size
-        )
+        return staticfiles_storage.url(f'img/addon-icons/default-{size}.png')
```

Comment:
does this mean we've dropped the cachebusting?  Or `staticfiles_storage.url` handles it for us?  (I assumed the latter but the tests below don't include any params?)

Reply:
The latter, but in tests we don't want to run collectstatic, so `STATICFILES_STORAGE` is not set to `ManifestStaticFilesStorage`, so there is no cachebusting. There is in prod (same for django's `static() templatetag, which calls `staticfiles_storage.url()` behind the scenes)